bot.js

```diff
@@ -629,11 +631,11 @@ function pretty_char2num(ch) {
     if (ch === ".") return -1;
     return "abcdefghjklmnopqrstuvwxyz".indexOf(ch.toLowerCase());
 }
-function move2gtpvertex(move, board_size) {
+function move2gtpvertex(move, width, height) {
```

Comment:
why do you pass `width` even though you dont use it in the function?
same question as https://github.com/online-go/gtp2ogs/pull/271#discussion_r424265135

Reply:
I think it's a tad clearer and less likely to cause errors due to typos since all the previous code passed 'width' (and would silently cause errors if used like that).

Reply:
hmm i'd still not pass parameters i don't use, if other parts of the code are unclear it is those parts that should be made clearer, not adding extra confusion in a clean code i think

but i don't know, maybe @Dorus and @roy7 have better opinions

Reply:
I think it's best to pass both sizes consistently to all functions like @sanderland is doing even if some of the info isn't used internally by the function at this time